include/zmq.h

```diff
@@ -263,10 +263,12 @@ typedef struct zmq_msg_t
 #endif
 } zmq_msg_t;
 
-typedef void(zmq_free_fn) (void *data_, void *hint_);
+typedef void (zmq_free_fn) (void *data_, void *hint_);
```

Comment:
don't change this, it's a false positive in the formatter tool

Reply:
same for other prototypes

Reply:
Thanks I'll have a look. I normally do format on save so this slips in!

Reply:
Fixed